### PR TITLE
CSC-1908: Add DDL for utils schema objects to ucb_6.0 branch

### PR DESCRIPTION
**Pull request by lkvoong**

### CODE REVIEW

**Comment by jblowe** [comment]
I'm pretty sure most of the PAHMA functions defined here are not needed: they are created nightly by the "hierarchy refresh scripts":

https://github.com/cspace-deployment/Tools/blob/master/scripts/pahma/hierarchies/reinstallfunctions.sh

Indeed, I would suggest that these NOT become part of the regular utils schema BECAUSE they are managed by these scripts -- there's a danger than someone might think they are 'static' components that need to be maintained. 

Basically, everything in the pahma directory with 'hierarchy' in its name can be excluded.

**Comment by jblowe** [comment]
... And I think there is a fair amount of other cruft as well, e.g.

src/main/resources/db/postgresql/utils_schema/cinefiles/17-TB-webfarmimgs.sql

Not sure what the best way is to handle this, but I suggest we try not to move more useless code than is absolutely necessary.

**Comment by lkvoong** [comment]
There are a number of PAHMA hierarchy* functions that create a number of objects. If we exclude the functions, we should also exclude the objects (e.g. tables, indexes) created by the functions.

**Comment by lkvoong** [comment]
Re: cruft, I removed everything I could identify as cruft. If you know of anything else that could be removed, please let me know.

**Comment by lkvoong** [comment]
Removed extraneous functions/tables/reviews that are managed by scripts elsewhere.
UCJeps: 06-VW-summary_counts_vw.sql is used by a report.

**Comment by lkvoong** [comment]
Re-added BAMPFA view and PAHMA Hierarchy functions.
